Sources/DangerSwiftEda/GitFlow.swift

```diff
@@ -43,7 +43,7 @@ public struct GitFlow {
         
     }
     
-    public struct Configuration {
+    public final class Configuration {
```

Comment:
[ask]
`struct` should be good... is there any concern that needs to make it a class?

Reply:
To utilize self.ticketAddressResolver in the branchParsingMethod, it needs to be a reference type.